README.md

```diff
@@ -53,14 +52,47 @@ $ npm run build
 
 Rollup looks at the following file [src/index.js](src/index.js) to determine what it should include in the production bundle... as such only import and export things in this file that you want to be included, otherwise do not touch this file.
 
-### Testing Components 🧪
+### Testing Strategies 🧪
+
+This project uses jest and the react testing library for testing. We've decided to use Jest as it is a widely used testing tool and provides sufficient resources for testing a component library. The React Testing Library integrate provides useful helpers to further push the flexibility of the jest testing tool. Some example test cases include;
 
-This project uses jest for testing. To run tests simply run the following command
+- Testing components with multiple styles (ex. button component with different button styles) [src/components/ActionButton.test.js](src/components/ActionButton.test.js)
+- Components styles are correctly rendered (Dimensions, Color, Fontstyle, Fontsize, etc) line 12 [src/components/Banner.test.js](src/components/Banner.test.js)
+- Clickable components (ex. button component has an onclick prop) line 17 [src/components/Menu.test.js](src/components/Menu.test.js)
+- Rendering with different input props line 19 [src/components/TextField.test.js](src/components/TextField.testjs)
+
+To run tests simply use the following command in the root directory
 
 ```bash
 $ npm run test
 ```
 
+For more information on tools and resources refer to our wifi page https://github.com/DTS-STN/DECD-Design-System/wiki#testing-documentation
+
```

Comment:
I reviewed storyshots yesterday and I think we should implement it since it essentially create snapshots automatically from our stories. I am going to open a separate ticket to implement storyshots but just wanted to make note of it here